### ISSUE TITLE
CI error in get_info

### ISSUE THREAD

**Issue by almarklein** [vispy/vispy#2094]
```
WARNING: could not determine DPI
WARNING: could not connect to display 
WARNING: Could not load the Qt platform plugin "xcb" in "" even though it was found.
WARNING: This application failed to start because no Qt platform plugin could be initialized. Reinstalling the application may fix this problem.

Available platform plugins are: eglfs, minimal, minimalegl, offscreen, vnc, webgl, xcb.

/home/runner/work/_temp/dc10969a-dac7-497b-bd5a-4bb412376607.sh: line 10:  3915 Aborted                 (core dumped) python -c "import vispy; print(vispy.sys_info())"
Error: Process completed with exit code 134.
```

I think the CI gods don't want us to release.

**Comment by larsoner** [comment]
If you run the offending code with `LD_DEBUG=libs` you'll get a ton of messages, a few of the last of which will tell you some `libxcb*` file is missing. It's likely one of these:

https://github.com/mne-tools/mne-python/blob/8c9c078e38cc4c7d67aa4e4f8a9b19eb32bb075e/tools/circleci_dependencies.sh#L7-L8

**Comment by larsoner** [comment]
(that is assuming you're using Desktop Qt5, if you're using EGL or something then I have no idea)

**Comment by kmuehlbauer** [comment]
In CI we install these packages:

https://github.com/vispy/vispy/blob/main/ci/requirements/linux_full_deps_apt.txt

So maybe we are missing some of them. Could be interconnected with using the new ubuntu in GHA, but that's just guesswork.

**Comment by djhoese** [comment]
Have we identified what has changed between yesterday and today?

**Comment by djhoese** [comment]
I see this in a recent failing environment:

```
 Err:23 http://azure.archive.ubuntu.com/ubuntu focal-updates/main amd64 libpulse0 amd64 1:13.99.1-1ubuntu3.10
  404  Not Found [IP: 52.252.75.106 80]
Get:24 http://azure.archive.ubuntu.com/ubuntu focal/universe amd64 libsdl2-2.0-0 amd64 2.0.10+dfsg1-3 [407 kB]
Get:25 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 libvulkan-dev amd64 1.2.131.2-1 [482 kB]
Get:26 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 libxfixes-dev amd64 1:5.0.3-2 [11.4 kB]
Get:27 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 x11proto-input-dev all 2019.2-1ubuntu1 [2628 B]
Get:28 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 libxi-dev amd64 2:1.7.10-0ubuntu1 [187 kB]
Get:29 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 x11proto-randr-dev all 2019.2-1ubuntu1 [2620 B]
Get:30 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 libxrandr-dev amd64 2:1.5.2-0ubuntu1 [25.0 kB]
Get:31 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 libwayland-bin amd64 1.18.0-1 [20.3 kB]
Get:32 http://azure.archive.ubuntu.com/ubuntu focal/main amd64 libwayland-dev amd64 1.18.0-1 [64.6 kB]
Get:33 http://azure.archive.ubuntu.com/ubuntu focal/universe amd64 mesa-utils amd64 8.4.0-1build1 [34.2 kB]
E: Failed to fetch http://azure.archive.ubuntu.com/ubuntu/pool/main/p/pulseaudio/libpulse0_13.99.1-1ubuntu3.10_amd64.deb  404  Not Found [IP: 52.252.75.106 80]
Fetched 2478 kB in 0s (28.1 MB/s)
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?
```

Looks like libpulse couldn't be downloaded, but that's a pretty odd dependency to cause an OpenGL-based dump (I assume).

**Comment by almarklein** [comment]
> Have we identified what has changed between yesterday and today?

The last merged PR was all green. I just reran the CI for that PR and now it fails with the same error. So this is not because of something we changed, but because the CI env has changed ...

**Comment by djhoese** [comment]
Biggest conda changes I see:

From:

```
h5py                                  3.2.1  nompi_py37ha3df211_100
numpy                                1.20.3  py37h038b26d_1
```

To:

```
h5py                                  3.3.0  nompi_py37ha3df211_100
numpy                                1.21.0  py37h038b26d_0
```

All pip dependencies are the same and the base conda packages from what I can tell. The ubuntu image is the same.

**Comment by djhoese** [comment]
Fixed by #2096 (at least from what I see watching the CI logs)
